common/workunit/workunit.cpp

```diff
@@ -3027,6 +3028,7 @@ class CWorkUnitFactory : public CInterface, implements IWorkUnitFactory, impleme
                 sortElements(iter, sortOrder.get(), NULL, NULL, unknownAttributes, elements);
                 return conn.getClear();
             }
+            virtual unsigned getTotalElements() { throw MakeStringException(-1, "not implemented"); }
```

Comment:
Not sure how to implement and if the implementation makes sense. Should I count all of the elements returned from dali controlled by the xpath? That count will include the WUs not match the post filters. If I remove those WUs from the total, it will be the same as the "returned". 
As I know, we are moving the WUs to Cassandra. Is it possible that Cassandra will allow us to return a subset of WUs and the 'total' value? Should we implement this after we move to Cassandra?